worker/lease/fixture_test.go

```diff
@@ -115,8 +115,6 @@ func (fix *Fixture) RunTest(c *gc.C, test func(*lease.Manager, *testclock.Clock)
 		case <-testDone:
 		case <-timeout:
 			c.Errorf("test took >10s to complete")
-			// If we timed out here, then we need to timeout storeDone as well
```

Comment:
Seems legit.. Why removing?

Reply:
Setting it doesn't do anything, nobody uses it afterwards.